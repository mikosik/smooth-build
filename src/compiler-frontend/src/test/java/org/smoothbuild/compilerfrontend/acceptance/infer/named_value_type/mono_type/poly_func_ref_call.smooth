A myId(A a) = a;
myValue = myId(7);
