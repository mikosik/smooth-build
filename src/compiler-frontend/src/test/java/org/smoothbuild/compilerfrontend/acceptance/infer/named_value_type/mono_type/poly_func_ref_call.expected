A myId<A>(A a)
  = a<>;
Int myValue<>
  = myId<Int>(7);
